templates/channels/channel_configuration.haml

```diff
@@ -510,6 +510,47 @@
       %code= ip_address
 
 
+  -elif object.channel_type == 'FCM'
+    %h4
+      -trans "To use your Firebase Cloud Messaging channel you'll have to POST to the following URLs with the parameters below."
+
+    .clearfix
+      .config-value
+        .name
+          -trans "Name:"
+        .value {{ object.name }}
+      .config-value
+        .name
+          -trans "Address/Key:"
+        .value {{ object.address }}
+      .config-value
+        .name
+          -trans "Notification title:"
+        .value {{ object.config_json.FCM_TITLE }}
+
```

Comment:
The Android app, for example, will call directly to RapidPRO (register and receive), Firebase will be used as a way of communication between RapidPRO and the app, precisely because of its notification feature. 

On the Firebase side, we just generate the key. So, we add a new FCM channel in RapidPRO, which will generate the URLs to integrate with the app, the app will listen to Firebase to receive the messages from RapidPRO throught user identifier in the app and use the URLs to send messages and register contacts on RapidPRO.